packages/react-native/src/types/NotificationAndroid.ts

```diff
@@ -39,6 +39,13 @@ export interface NotificationAndroid {
    */
   asForegroundService?: boolean;
 
+  /**
+   * When set to `true` the screen will light up when the notification be displayed.
```

Comment:
tiny grammar nitpick - nothing more fun than how irregular all the "to be" verb forms are in English (or Spanish, or Portuguese I bet ;-) )

```suggestion
   * When set to `true` the screen will light up when the notification is displayed.
```

Reply:
Portuguese is the same haha (a lot of different patterns).
I'll fix it, thanks!